### PR TITLE
Fixes #263 :bug: [AI] Multi trainner

### LINKED ISSUE #263: AI Worker : script installation have 2 VM ready
We are stuck because of multiple parallele trainings that need to be done, the task / job model let's us do that but we need one more trainer (VM for that) to be able to process more jobs in parallel also we need a clean way to recover in case the trainer VM breaks.

We should script the installation of donkeycar using Ansible for a paperspace basic ubuntu VM (P4000).

### PR DESCRIPTION
**Pull request by Benvii**
Fixes #263 multiple AI trainner

### CODE REVIEW

**Comment by Benvii** [comment]
Tested today it works well but we still need to improve FTP transfert resiliency #264

**Comment by Benvii** [comment]
I someone want's to review it, @viandoxdev  and/or @yannis-mlgrn

File: dkmanager_trainner/dkmanager_trainer/services/ai_trainer_worker_service.py [hunk 1]
```diff
@@ -34,17 +48,45 @@ def __init__(self, api_origin: Optional[str] = None,
...
-        ai_workers = self._api.get_workers(worker_type=WorkerType.AI_TRAINER)
+        worker = None
+
+        ai_worker_config = AiTrainerWorkerService.get_ai_worker_config()
+        if ai_worker_config is not None:
+            worker = self._api.get_worker(ai_worker_config.worker_id)
 
-        if len(ai_workers) > 0: # We assume there is only one IA Worker at all
-            worker = ai_workers[0]
+        if worker is not None:  # We assume there is only one IA Worker at all
```
Comment (viandoxdev) [inline dkmanager_trainner/dkmanager_trainer/services/ai_trainer_worker_service.py:87]: Isn't this comment wrong ?

**Review by viandoxdev — APPROVED** [review]
LGTM, just one comment I'm not sure I understand

File: dkmanager_trainner/dkmanager_trainer/services/ai_trainer_worker_service.py (hunk 1 above)
Comment (Benvii) [reply dkmanager_trainner/dkmanager_trainer/services/ai_trainer_worker_service.py]: Comment it wrong yes,  updated here : 88c36e4

**Pull request merged**